src/redux/middlewares.js

```diff
@@ -111,12 +111,12 @@ export const scrollMiddleware = store => next => action => {
 import {init, scrollCompensator} from '../services/realtime'
 
 const bindHandlers = dispatch => ({
-  'post:new': scrollCompensator(data => dispatch({type: ActionTypes.REALTIME_POST_NEW, post : data.posts})),
-  'post:update': scrollCompensator(data => dispatch({type: ActionTypes.REALTIME_POST_UPDATE, post : data.posts})),
+  'post:new': scrollCompensator(data => dispatch({type: ActionTypes.REALTIME_POST_NEW, post : data.posts, ...data})),
+  'post:update': scrollCompensator(data => dispatch({type: ActionTypes.REALTIME_POST_UPDATE, post : data.posts, ...data})),
   'post:destroy': scrollCompensator(data => dispatch({type: ActionTypes.REALTIME_POST_DESTROY, postId: data.meta.postId})),
   'post:hide': scrollCompensator(data => dispatch({type: ActionTypes.REALTIME_POST_HIDE, postId: data.meta.postId})),
   'post:unhide': scrollCompensator(data => dispatch({type: ActionTypes.REALTIME_POST_UNHIDE, postId: data.meta.postId})),
-  'comment:new': scrollCompensator(data => dispatch({type: ActionTypes.REALTIME_COMMENT_NEW, comment: data.comments})),
+  'comment:new': scrollCompensator(data => dispatch({type: ActionTypes.REALTIME_COMMENT_NEW, comment: data.comments, ...data})),
```

Comment:
Why the `data` has priority here? What if, say, `data` has property `comment`, and it will override previous assignment?

Is it not `{...data, type: ActionTypes.REALTIME_COMMENT_NEW, comment: data.comments}` on purpose?